docs/setup.md

```diff
@@ -5,16 +5,16 @@
 
 1. To install a widget, click **Content Hub** > **Discover**.
 
-2. From the list of solution pack that appears, search for and select **Global Visibility**. 
+2. From the list of widgets that appears, search for and select **Global Visibility**. 
 
-3. Click the card of the **Global Visibility** solution pack card.
+3. Click the **Global Visibility** widget card to begin installation.
```

Comment:
No use of this widget installed solely, It is meant for global visibility solution pack. And will be installed along with SP. We can remove the Installation Tag above

Same for the 2nd point too

Reply:
Even if the widget is part of an SP, we still document its installation process. For consistency and if in future, it evolves to be something independent.